admin/app/adapters/certification.js

```diff
@@ -5,24 +5,27 @@ export default class Certification extends ApplicationAdapter {
     return `${this.host}/${this.namespace}/admin/certifications/${id}`;
   }
 
-  urlForUpdateMarks(id) {
-    return `${this.host}/${this.namespace}/admin/certification-courses/${id}/assessment-results/`;
+  urlForUpdateComments(id) {
+    return `${this.host}/${this.namespace}/admin/certification-courses/${id}/assessment-results`;
   }
 
   urlForUpdateRecord(id) {
     return `${this.host}/${this.namespace}/certification-courses/${id}`;
   }
 
   updateRecord(store, type, snapshot) {
-    const data = {};
-    const serializer = store.serializerFor(type.modelName);
-    if (snapshot.adapterOptions.updateMarks) {
-      serializer.serializeIntoHash(data, type, snapshot, { includeId: true });
-      data.data.type = 'results';
-      data.data.attributes['jury-id'] = null;
-      data.data.attributes['emitter'] = 'Jury Pix';
-      return this.ajax(this.urlForUpdateMarks(snapshot.id), 'POST', { data: data });
+    if (snapshot.adapterOptions.updateComments) {
+      const payload = this.serialize(snapshot);
+      payload.data.attributes = {
+        'comment-for-organization': payload.data.attributes['comment-for-organization'],
+        'comment-for-candidate': payload.data.attributes['comment-for-candidate'],
+        'comment-by-jury': payload.data.attributes['comment-by-jury'],
+      };
+      delete payload.data.relationships;
```

Comment:
```suggestion
      const { data: { attributes, type }} = this.serialize(snapshot);
      const payload = {
        data: {
          attributes: {
            'comment-for-organization': attributes['comment-for-organization'],
            'comment-for-candidate': attributes['comment-for-candidate'],
            'comment-by-jury': attributes['comment-by-jury'],
          },
          type,
        },
      };
```
Je propose ça sur la base de la validation JOI sur la route associée. J'ai l'impression d'avoir mis tout ce qui est nécessaire et que c'est plus lisible

Reply:
Joi peut être permissif, on passe peut-etre plus de donnée que joi ne le montre. Si c'est ok apres tests, c'est plus simple et lisible qu'avec le delete.

Reply:
ça fonctionne, je fais la modification

Reply:
Done